src/modules/menu/components/share/MenuShareShortLinkInput.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+    <div v-if="shortLink" class="p-1">
+        <label v-if="withText">{{ $t('share_link') }}: </label>
+        <div class="input-group input-group-sm">
+            <input
+                type="text"
+                class="form-control"
+                readonly="readonly"
+                :value="shortLink"
+                @focus="$event.target.select()"
+            />
+            <button class="btn btn-outline-secondary" @click="copyShortLinkInClipboard">
+                {{ $t(copiedInClipboard ? 'copy_success' : 'copy_url') }}
+            </button>
+        </div>
+    </div>
+</template>
+
+<script>
+export default {
+    props: {
+        shortLink: {
+            type: String,
+            default: null,
+        },
+        withText: {
+            type: Boolean,
+            default: true,
+        },
+    },
+    data() {
+        return {
+            copiedInClipboard: false,
+            timeoutCopied: null,
+        }
+    },
+    watch: {
+        shortLink() {
+            this.clearIsCopiedInClipboard()
+        },
+    },
+    methods: {
+        clearIsCopiedInClipboard() {
+            this.copiedInClipboard = false
+            if (this.timeoutCopied) {
+                clearTimeout(this.timeoutCopied)
+                this.timeoutCopied = null
+            }
+        },
+        copyShortLinkInClipboard() {
+            navigator.clipboard.writeText(this.shortLink)
```

Comment:
Did you test this in Firefox?
Mine (v99.0) doesn't have `navigator.clipboard`.

Reply:
maybe I should use this signature instead : https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/writeText
but it should be supported across all browser : https://caniuse.com/mdn-api_clipboard_writetext

Reply:
well, in the exemple given on the mozilla webpage, it's the exact same signature... With your version of Firefox it should be supported, weird...

Reply:
Sorry! My bad. I tend to use about:blank to test browser APIs and it seems that clipboard is not available there.

Reply:
nope, you need to be either in HTTPS or on localhost for this API to be enabled 😉